opentech/apply/projects/templates/application_projects/project_detail.html

```diff
@@ -0,0 +1,154 @@
+{% extends "base-apply.html" %}
+
+{% load static  %}
+
+{% block title %}{{ object.title }}{% endblock %}
+
+{% block body_class %}{% endblock %}
+
+{% block content %}
+<div class="admin-bar">
+    <div class="admin-bar__inner">
+        <h1 class="beta heading heading--no-margin heading--bold">{{ object.name }}</h1>
+        {# {% status_bar object.workflow object.phase request.user same_stage=True%} #}
+
+        <div class="tabs js-tabs">
+            <div class="tabs__container">
+                <a class="tab__item" href="#details" data-tab="tab-1">
+                    Details
+                </a>
+
+                <a class="tab__item" href="#communications" data-tab="tab-2">
+                    Communications
+                </a>
+
+                <a class="tab__item" href="#activity-feed" data-tab="tab-3">
+                    Activity Feed
+                </a>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="wrapper wrapper--large wrapper--tabs js-tabs-content">
+    <div class="tabs__content" id="tab-1">
+
+        <div class="wrapper wrapper--sidebar">
+            <article class="wrapper--sidebar--inner">
+                <header class="heading heading--submission-meta heading-text zeta">
+                    <span>Last edit: <strong>{{ object.updated_at.timestamp.date }} by {{ object.updated_by }}</strong></span>
+
+                    {# {% if request.user|has_edit_perm:object %} #}
+                    {% if request.user %}
+                    <a class="link link--edit-submission is-active" href="#">
+                        Edit
+                        <svg class="icon icon--pen"><use xlink:href="#pen"></use></svg>
+                    </a>
+                    {% else %}
+                    <span class="link link--edit-submission">
+                        Edit
+                        <svg class="icon icon--padlock"><use xlink:href="#padlock"></use></svg>
+                    </span>
+                    {% endif %}
+                </header>
+
+                <h3>Project Information</h3>
+                <div class="grid grid--proposal-info">
+                    <div>
+                        <h5>Proposed start date</h5>
+                        <p>13 August 2019</p>
+                    </div>
+
+                    <div>
+                        <h5>Project Proposed end date</h5>
+                        <p>25 January 2020</p>
+                    </div>
+
+                    <div>
+                        <h5>Legal name</h5>
+                        <p>The Company</p>
```

Comment:
Can you hook this up to the model please.

Reply:
Fixed in b6a88dfb